src/transaction/boot_cl.c

```diff
@@ -1305,6 +1305,13 @@ boot_restart_client (BOOT_CLIENT_CREDENTIAL * client_credential)
     }
   json_set_alloc_funcs (malloc, free);
 
+  /* Initialize of the lzo library. */
+  error_code = lzo_init ();
+  if (error_code != LZO_E_OK)
+    {
+      goto error;
+    }
+
```

Comment:
@lucianlazar2211 

We may have to examine what will happen for `SA_MODE`.
`lzo_init` will be called twice for `SA_MODE`, one for client and the other for server.